test/WebJobs.Script.Tests/TestScripts/Node/functions.tests.js

```diff
@@ -99,9 +128,9 @@ describe('http', () => {
         });
 
         it('get', () => {
-            expect(req.get('test')).to.equal('val');
```

Comment:
IMO it's not necessary in a simple case like this when uppercase is a superset of lowercase

However let me know if you really want the extra test and I'll add